resources/levels-progression/recursion.json

```diff
@@ -12,6 +12,17 @@
         "sun": "__variant_weekday_sun"
     },
     "levels": [
+        {
```

Comment:
Is this a debug level?

Reply:
Ah yes. Should remove unless we want to use it tomorrow to demo this feature?